host_guest/SAMPLing/absolute_submission_example.txt

```diff
@@ -0,0 +1,88 @@
+# SAMPLing challenge results
+#
+# This file will be automatically parsed. It must contain the following five sections: predictions, computational cost,
+# software listing, technical details, and method description. These elements must be provided in the order shown.
+#
+# The data in each prediction line should be a comma-separated list of exactly 201 values, 100 of which are optional.
+# Each line must be structured as follows: system ID, F(1%), dF(1%), F(2%), dF(2%), ..., F(100%), dF(100%), where
+# F(X%) and dF(X%) are respectively the free energy estimate and free energy uncertainty, both in kcal/mol, after X% of
```

Comment:
Do you want to say something here about which hosts/guests should be in the file? Right now this is not clear.

Remember, the SAMPLing challenge formally consists of just a couple systems. If people want to submit values for more, we are happy to accept them (and, maybe, curate them/report on them -- or at least just serve to certify that yes, they got these values fairly) but most will be optional.

Reply:
I thought this information would go in the instructions, but I can definitely add a note here as well!

I'll make the suggested modifications and add the example for relative free energy calculations.